src/playground-ide.ts

```diff
@@ -21,7 +21,7 @@ import './playground-file-editor.js';
 import './playground-preview.js';
 import {PlaygroundProject} from './playground-project.js';
 import {ProjectManifest} from './shared/worker-api.js';
-import {forceSkypackRawMode} from './shared/util.js';
```

Comment:
For you use the fn anywhere else? If not don't forget to delete it

Reply:
Yep, it's also used in `playground-project.ts` to resolve the typescript web worker when imported from skypack.